libs/libvtrutil/src/vtr_ragged_matrix.h

```diff
@@ -0,0 +1,188 @@
+#ifndef VTR_RAGGED_MATRIX_H
```

Comment:
This needs unit tests in libs/libvtrutils/test

Reply:
Done